### ISSUE TITLE
DataBase Integration

### ISSUE THREAD

**Issue by minedun6** [fxpresso/FXMaps#9]
I was wondering if there might be a way to put the coordinates in a mysql database.
The library is really awesome but still misses the most import part which is the documenation.
I have an academic project I'd like to use this library for if it's not a problem with you.

**Comment by cogmission** [comment]
@minedun6 I'm sorry, the library is not at its first point of usability yet. The API still needs rounding out (i.e. needs to have the basics). I need to put another week's work into it and then it will be ready for general use (but not near total completion). DB integration will be up to you even though. You will have to write your Routes to the DB. Right now, it writes to a local file, you can take a look to see how it does that  via the MapStore.class file (see: PersistentMap also). Sorry...

**Comment by minedun6** [comment]
Yep, I did struggle a bit to found it but it's ok, all I'm gonna be needing is saving marker location that i add into the DB, another thing i need to point, i tried the demo and one thing that jumps to my mind is the fact that I can't add other markers to my routes once i reload it. So u did it intentionally or maybe u didn't notice that ?
